### ISSUE TITLE
Suppress automatic sentence segmentation

### ISSUE THREAD

**Issue by jerrybonnell** [megagonlabs/ginza#153]
Hi, we have been using `ginza` in research on corpus development. Since version 4 it looks like the project has introduced a sentence segmentation mode where the raw sequence can be split into multiple sentences. This is an example with ginza 4.0.4:

```
echo "大に従来の事業を革新拡張せんとするものは" | ginza

# text = 大に
1	大に	大に	ADV	副詞	_	0	root	_	SpaceAfter=No|BunsetuBILabel=B|BunsetuPositionType=ROOT|Reading=オオイニ

# text = 従来の事業を革新拡張せんとするものは
1	従来	従来	NOUN	名詞-普通名詞-副詞可能	_	3	nmod	_SpaceAfter=No|BunsetuBILabel=B|BunsetuPositionType=SEM_HEAD|NP_I|Reading=ジュウライ
2	の	の	ADP	助詞-格助詞	_	1	case	_	SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SYN_HEAD|Reading=ノ
3	事業	事業	NOUN	名詞-普通名詞-一般	_	6	obj	_SpaceAfter=No|BunsetuBILabel=B|BunsetuPositionType=SEM_HEAD|NP_I|Reading=ジギョウ
4	を	を	ADP	助詞-格助詞	_	3	case	_	SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SYN_HEAD|Reading=ヲ
5	革新	革新	NOUN	名詞-普通名詞-サ変可能	_	6	compound	SpaceAfter=No|BunsetuBILabel=B|BunsetuPositionType=CONT|Reading=カクシン
6	拡張	拡張	VERB	名詞-普通名詞-サ変可能	_	10	advcl	_SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SEM_HEAD|Reading=カクチョウ
7	せ	する	AUX	動詞-非自立可能	_	6	advcl	_	SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SYN_HEAD|Inf=サ行変格,未然形-セ|Reading=セ
8	ん	ぬ	AUX	助動詞	_	6	aux	_	SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SYN_HEAD|Inf=助動詞-ヌ,終止形-撥音便|Reading=ン
9	と	と	ADP	助詞-格助詞	_	6	case	_	SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SYN_HEAD|Reading=ト
10	する	する	VERB	動詞-非自立可能	_	11	acl	_	SpaceAfter=No|BunsetuBILabel=B|BunsetuPositionType=SEM_HEAD|Inf=サ行変格,連体形-一般|Reading=スル
11	もの	もの	NOUN	名詞-普通名詞-サ変可能	_	0	root	_SpaceAfter=No|BunsetuBILabel=B|BunsetuPositionType=ROOT|NP_I|Reading=モノ
12	は	は	ADP	助詞-係助詞	_	11	case	_	SpaceAfter=No|BunsetuBILabel=I|BunsetuPositionType=SYN_HEAD|Reading=ハ
```
Please see how `大に` is treated as a new sentence. I wonder if there is any way to suppress the sentence segmentation so that the sequence is treated as a single sentence only? It is important for us since our corpus is already annotated with sentences.  

日本語で話せばもっとよければ、教えてください。ありがとう！

**Comment by hiroshi-matsuda-rit** [comment]
Sorry for waiting. For the spaCy v2.x, we need to set `token.is_sent_start = False` for all the tokens in a doc before the parser in pipeline. https://spacy.io/usage/linguistic-features#sbd-custom
I'd like to add `--disable-sentencizer` option to `command_line.py` soon.

**Comment by hiroshi-matsuda-rit** [comment]
I just released GiNZA v4.0.5. You can use -d option of command line tool to disable spaCy's sentencizer by specifying -d option like:

`$ ginza -d`
